src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/loader/JdbcConnectionLoader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.loader;
+
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.JdbcConnection;
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Service;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Service'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=53&issues=AWU6Y48iWWqqPQVjPd10&open=AWU6Y48iWWqqPQVjPd10)

[ISSUE_KEY]: AWU6Y48iWWqqPQVjPd10